lib/iris/analysis/maths.py

```diff
@@ -342,14 +340,6 @@ def _add_subtract_common(operation_function, operation_name, cube, other,
         # operation with
         coord_comp = iris.analysis.coord_comparison(cube, other)
 
-        # provide a deprecation warning if the ignore keyword has been set
-        if ignore is not True:
-            msg = ('The "ignore" keyword has been deprecated in '
-                   'add/subtract. This functionality is now automatic. '
-                   'The provided value to "ignore" has been ignored, '
-                   'and has been automatically calculated.')
-            warn_deprecated(msg)
```

Comment:
Now that you've removed this usage of `warn_deprecated` you'll also need to remove the import on [L34](https://github.com/lbdreyer/iris/blob/755fb3da11633443a86e1d1375004f1b36b0741d/lib/iris/analysis/maths.py#L34).